rubygem-hammer_cli_sam/rubygem-hammer_cli_sam.spec

```diff
@@ -0,0 +1,85 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name hammer_cli_sam
+%global confdir hammer
+
+Summary: SAM commands for Hammer
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.0.1
+Release: 1%{?dist}
+Group:   Development/Languages
+License: GPLv3
+URL:     http://github.com/Katello/hammer-cli-sam
+Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+%if 0%{?scl:1}
+Obsoletes: rubygem-%{gem_name} < 1.0.1-2
+%endif
+
+%if 0%{?fedora} > 18
+Requires:  %{?scl_prefix}ruby(release)
+%else
+Requires:  %{?scl_prefix}ruby(abi)
+%endif
+Requires: %{?scl_prefix}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(hammer_cli) >= 0.1.1
```

Comment:
I would think this needs to depend on `hammer_cli_katello`, not plain `hammer_cli`?